File: app/src/main/java/com/zac4j/browser/MainActivity.java
package com.zac4j.browser;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import com.zac4j.browser.util.system.PermsSettingJumper;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void pickImage(View view) {
        if (view != null) {
            //startActivity(new Intent(MainActivity.this, PickerActivity.class));
            PermsSettingJumper.goToPermsSettingPage(MainActivity.this);
        }
    }
}

File: app/src/main/java/com/zac4j/browser/ui/MainActivity.kt
package com.zac4j.browser.ui

import android.os.Bundle
import android.support.design.widget.BottomNavigationView
import android.support.v7.app.AppCompatActivity
import androidx.navigation.findNavController
import androidx.navigation.ui.setupActionBarWithNavController
import androidx.navigation.ui.setupWithNavController
import com.zac4j.browser.R

/**
 * Created by Zaccc on 2018/8/24.
 */
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val bottomNavigationView = findViewById<BottomNavigationView>(R.id.main_nav_bottom)
        val navController = findNavController(R.id.main_nav_fragment)
        setupActionBarWithNavController(navController)
        bottomNavigationView.setupWithNavController(navController)
    }

    override fun onSupportNavigateUp() = findNavController(R.id.main_nav_fragment).navigateUp()
}